### PR TITLE
Add new Checkstyle and ESLint rules - line length 130

### PR DESCRIPTION
**Pull request by wkurniawan07**
Still quite some time before reaching <= 120.

### CODE REVIEW

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java [hunk 1]
```diff
@@ -568,8 +577,12 @@ public String getCsvHeader() {
...
     public String getQuestionTypeChoiceOption() {
         // Constant sum has two options for user to select.
-        return "<li data-questiontype = \"CONSTSUM_OPTION\"><a>" + Const.FeedbackQuestionTypeNames.CONSTSUM_OPTION + "</a></li>"
-               + "<li data-questiontype = \"CONSTSUM_RECIPIENT\"><a>" + Const.FeedbackQuestionTypeNames.CONSTSUM_RECIPIENT + "</a></li>";
+        return "<li data-questiontype = \"CONSTSUM_OPTION\">"
+                 + "<a>" + Const.FeedbackQuestionTypeNames.CONSTSUM_OPTION + "</a>"
+             + "</li>"
+             + "<li data-questiontype = \"CONSTSUM_RECIPIENT\">"
+                 + "<a>" + Const.FeedbackQuestionTypeNames.CONSTSUM_RECIPIENT + "</a>"
+             + "</li>";
```
Comment (YongJieYongJie) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: Some indentation problems here

File: src/main/java/teammates/common/util/Const.java [hunk 2]
```diff
@@ -1274,14 +1323,16 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String INSTRUCTOR_NO_MODIFY_PERMISSION_FOR_ACTIVE_COURSES_SESSIONS =
                                    "No permission to modify any sessions in un-archived courses";
```
Comment (YongJieYongJie) [inline src/main/java/teammates/common/util/Const.java]: Maybe this needs to be indented at 8 spaces for consistency

File: src/main/java/teammates/common/util/EmailLogEntry.java [hunk 3]
```diff
@@ -54,18 +54,31 @@ public EmailLogEntry(AppLogLine appLog) {
...
+                 + "<td colspan=\"3\">"
+                     + "<div class=\"well\">"
+                         + "<ul class=\"list-group\">"
+                             + "<li class=\"list-group-item list-group-item-success\">"
+                                 + "<small>" + this.content + "</small>"
+                             + "</li>"
+                         + "</ul>"
+                     + "</div>"
+                 + "</td>"
+             + "</tr>";
```
Comment (YongJieYongJie) [inline src/main/java/teammates/common/util/EmailLogEntry.java]: Some indentation problem after the first line with the word `return`

File: src/main/java/teammates/logic/core/CommentsLogic.java [hunk 4]
```diff
@@ -405,7 +405,8 @@ private void removeNonVisibleCommentsForTeam(List<CommentAttributes> commentsFor
                     preventAppendingThisCommentAgain(commentsVisitedSet, c);
                 }
             //for team
-            } else if (c.recipientType == CommentParticipantType.TEAM && c.recipients.contains(Sanitizer.sanitizeForHtml(student.team))) {
+            } else if (c.recipientType == CommentParticipantType.TEAM
+                    && c.recipients.contains(Sanitizer.sanitizeForHtml(student.team))) {
```
Comment (YongJieYongJie) [inline src/main/java/teammates/logic/core/CommentsLogic.java]: Indentation error from the `&&` onwards

File: src/main/java/teammates/logic/core/FeedbackResponsesLogic.java [hunk 5]
```diff
@@ -437,7 +437,8 @@ public void updateFeedbackResponse(
             newResponse.recipientSection = oldResponse.recipientSection;
         }
     
-        if (newResponse.recipientEmail.equals(oldResponse.recipientEmail) && newResponse.giverEmail.equals(oldResponse.giverEmail)) {
+        if (newResponse.recipientEmail.equals(oldResponse.recipientEmail)
+                && newResponse.giverEmail.equals(oldResponse.giverEmail)) {
```
Comment (YongJieYongJie) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: Maybe can align the second line with `[n]ewResponse`

File: src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java [hunk 6]
```diff
@@ -197,7 +202,8 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToAdd,
                     String valueForSectionName =
                             getRequestParamValue(Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + i
                                                  + Const.ParamsNames.INSTRUCTOR_SECTION + j);
-                    if (isSectionGroupSpecial && valueForSectionName != null && sectionNamesTable.containsKey(valueForSectionName)) {
+                    if (isSectionGroupSpecial && valueForSectionName != null
+                            && sectionNamesTable.containsKey(valueForSectionName)) {
```
Comment (YongJieYongJie) [inline src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java]: The 4 spaces indentation seems to suggest that there are only two conditionals when there are in fact 3. Perhaps align the `&&` with `[i]sSectionGroupSpecial`

File: src/main/java/teammates/ui/controller/InstructorCourseInstructorDeleteAction.java [hunk 7]
```diff
@@ -56,17 +57,19 @@ protected ActionResult execute() {
...
 
         for (InstructorAttributes instr : instructors) {
 
-            boolean isAlternativeInstructor = instr.isRegistered()
-                                              && !instr.getEmail().equals(instructorToDeleteEmail)
-                                              && instr.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR);
+            boolean isAlternativeInstructor =
+                    instr.isRegistered()
+                    && !instr.getEmail().equals(instructorToDeleteEmail)
+                    && instr.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR);
```
Comment (YongJieYongJie) [inline src/main/java/teammates/ui/controller/InstructorCourseInstructorDeleteAction.java]: Perhaps indent `instr.isRegistered()` onwards 4 more spaces

File: src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java [hunk 8]
```diff
@@ -221,7 +228,8 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToEdit,
                     String valueForSectionName =
                             getRequestParamValue(Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + i
                                                  + Const.ParamsNames.INSTRUCTOR_SECTION + j);
-                    if (isSectionGroupSpecial && valueForSectionName != null && sectionNamesTable.containsKey(valueForSectionName)) {
+                    if (isSectionGroupSpecial && valueForSectionName != null
+                            && sectionNamesTable.containsKey(valueForSectionName)) {
```
Comment (YongJieYongJie) [inline src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java]: Same as somewhere above:
the indented `&&` suggests only two conditionals when there are in fact three, maybe don't indent the second line

File: src/test/java/teammates/test/cases/ui/InstructorEditInstructorFeedbackPageActionTest.java [hunk 9]
```diff
@@ -74,11 +75,12 @@ public void testExecuteAndPostProcess() {
...
-                + "Moderating feedback session for instructor (" + moderatedInstructor.email + ")<br>"
-                + "Session Name: Another feedback session<br>Course ID: IEIFPTCourse|||"
-                + "/page/instructorEditInstructorFeedbackPage",
+        AssertHelper.assertLogMessageEquals(
+                "TEAMMATESLOG|||instructorEditInstructorFeedbackPage|||instructorEditInstructorFeedbackPage"
+                    + "|||true|||Instructor|||IEIFPTCourseinstr|||IEIFPTCourseinstr|||IEIFPTCourseintr@course1.tmt|||"
+                    + "Moderating feedback session for instructor (" + moderatedInstructor.email + ")<br>"
+                    + "Session Name: Another feedback session<br>Course ID: IEIFPTCourse|||"
+                    + "/page/instructorEditInstructorFeedbackPage",
                 editInstructorFpAction.getLogMessage());
```
Comment (YongJieYongJie) [inline src/test/java/teammates/test/cases/ui/InstructorEditInstructorFeedbackPageActionTest.java]: Inconsistency: the lines with `+` are indented 4 spaces to separate the first and second arguments, but elsewhere, 8 spaces are used (e.g., below at `InstructorEditStudentFeedbackPageActionTest.java:150`

File: src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java [hunk 10]
```diff
@@ -426,7 +426,9 @@ public void testDeleteQuestionAction() {
 
     private InstructorFeedbackEditPage getFeedbackEditPage() {
         AppUrl feedbackPageLink = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE)
-                                        .withUserId(instructorId).withCourseId(instructorCourseId).withSessionName(instructorEditFSName);
+                .withUserId(instructorId)
+                .withCourseId(instructorCourseId)
+                .withSessionName(instructorEditFSName);
```
Comment (YongJieYongJie) [inline src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java]: Since these lines are short, maybe indent wrt `createUrl`

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Added some comments

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java (hunk 1 above)
Comment (wkurniawan07) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: This one's to show the HTML hierarchy.

File: src/main/java/teammates/common/util/Const.java (hunk 2 above)
Comment (wkurniawan07) [inline src/main/java/teammates/common/util/Const.java]: I don't touch this line... -.-
(but sure will do)

File: src/main/java/teammates/common/util/EmailLogEntry.java (hunk 3 above)
Comment (wkurniawan07) [inline src/main/java/teammates/common/util/EmailLogEntry.java]: Copying above: This one's to show the HTML hierarchy.

File: src/main/java/teammates/logic/core/FeedbackResponsesLogic.java (hunk 5 above)
Comment (wkurniawan07) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: Nope, the minimum needed is 8 spaces after `if`.

File: src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java (hunk 6 above)
Comment (wkurniawan07) [inline src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java]: Refer to the comment above.

File: src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java (hunk 8 above)
Comment (wkurniawan07) [inline src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java]: Again, refer to the comment above.

File: src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java (hunk 10 above)
Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java]: Look at all the similar lines after this: they're not indented wrt `createUrl`.

Comment (YongJieYongJie) [inline src/test/java/teammates/test/cases/ui/browsertests/FeedbackRankQuestionUiTest.java]: I see I see, then maybe indent wrt to `feedbackPageLink`

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Besides the `FeedbackRankQuestionUiTest.java` everything looks fine!

**Comment by wkurniawan07** [comment]
@YongJieYongJie done!

**Comment by damithc** [comment]
This is a big one :-p. Kudos :+1:

**Pull request merged**